config/clusters/fyre/Chart.yaml

```diff
@@ -16,9 +16,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.0.1
```

Comment:
the fyre configuration is only for demo purpose, it does not need to be kept in sync w/ AIOps version.